third_party/proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -13,28 +14,17 @@ option (gogoproto.goproto_getters_all) = false;
 // GenericAuthorization gives the grantee unrestricted permissions to execute
 // the provided method on behalf of the granter's account.
 message GenericAuthorization {
-  option (cosmos_proto.implements_interface) = "Authorization";
+  option (amino.name)                        = "cosmos-sdk/GenericAuthorization";
```

Comment:
Note to self: Investigate what this `amino.name` means/does (and if we need to add it to some of our stuff).